app/events/events.go

```diff
@@ -229,9 +238,7 @@ func transform(msg *tbapi.Message) *bot.Message {
 		Text: msg.Text,
 	}
 
-	if msg.Chat != nil {
-		message.ChatID = msg.Chat.ID
-	}
+	message.ChatID = msg.Chat.ID
```

Comment:
any good reason to remove safety nil check?

Reply:
It's not a pointer anymore so it would be set anyway. I presume it might be zero, but it will be there all the time.